docs: [DHIS2-11208][DHIS2-11402][DHIS2-11199][DHIS2-11201] enrollment edit event page (#788)
dhis2/dhis2-docs: src/user/using-the-capture-app.md

## Enrollment event view and edit page

### Navigation 

You can reach the enrollment event edit page is by clicking any event in the Program stage list from the Enrollment Overview page.

### Top bar context 

> At the top of the page you can see different informations related with the current context. You can see the program, the organization unit, the tracked entity, the enrollment date, the stage and the enrollmenet event date.

Reviewer: ```suggestion
At the top of the page you can see various information related to the current context. You can see the program, the organization unit, the tracked entity, the enrollment date, the stage and the enrollment event date.
```

In the view/edit event form you can see the stage name and icon.

![](resources/images/capture_app/enrollment-event-view-edit-header.png)

#### View mode

> When the form is in the edit mode the title of the page will appear as: `Enrollment: View Event`. You can see in the page all the informations related with the event. Click the `Edit event` button to switch to the edit mode. 

Reviewer: ```suggestion
When the form is in the view mode the title of the page will appear as: `Enrollment: View Event`. You can see in the page all the information related to the event. Click the `Edit event` button to switch to the edit mode. 
```
